infra/gcb/build_and_run_coverage.py

```diff
@@ -32,6 +32,14 @@
 UPLOAD_REPORT_URL_FORMAT = 'gs://%s/{0}/reports/{1}' % COVERAGE_BUCKET_NAME
 
 
+def skip_build(message):
+  # Exit with 0 not to mark code coverage job as failed.
+  # Since the script should print build_id, print '0' as a special value.
+  sys.stderr.write('%s\n' % message)
+  print '0'
```

Comment:
Is this print needed now ?

Reply:
Yes, it's still needed, at least while we're still using Jenkins with `wait_for_build.py` helper script.